.github/workflows/latest.yml

```diff
@@ -52,15 +57,22 @@ jobs:
           ${{ runner.os }}-golang-
     - uses: chainguard-dev/actions/setup-kind@main
       with:
-        k8s-version: v1.24.x
+        k8s-version: v1.25.x
     - uses: vdemeester/setup-tektoncd@main
       with:
-        pipeline: v0.49.x
+        pipeline: v0.51.x
         pipeline-feature-flags: '{"enable-api-fields": "beta"}'
+    - uses: docker/login-action@v3
+      with:
+        registry: ghcr.io
+        username: ${{ github.actor }}
+        password: ${{ secrets.GITHUB_TOKEN }}
     - name: install manual-approval-gate custom task
       run: |
         # kubectl create namespace tekton-pipelines
-        ko apply --local -f config/kubernetes
+        ko apply -f config/kubernetes
```

Comment:
you would need to use ` --base-import-paths` for this, as well as, probably using the `commit` sha as a tag or something to make sure we don't override `:latest`.